erdpy_network/network_config.py

```diff
@@ -14,6 +14,7 @@ def __init__(self) -> None:
         self.min_gas_price: int = 1000000000
         self.gas_price_modifier: float = 1
         self.min_transaction_version: int = 1
+        self.num_shards_without_meta: int = 0
```

Comment:
might use `3` as default?

Reply:
I don't think it's really relevant. The object usually get initialized right away, so the default value does not matter. And if the number of shards modifies we'll have to change the default value as well.

Reply:
The min gas price can also be changed and we'd have to update the constant. It's just for consistency, since the other values seem to be initialized with the current correct values

Reply:
I agree that the consistency is good. Changed the value to `3`.